service/create/service.go

```diff
@@ -298,10 +313,11 @@ type runMachinesInput struct {
 	prefix              string
 }
 
-func (s *Service) runMachines(input runMachinesInput) error {
+func (s *Service) runMachines(input runMachinesInput) ([]*string, error) {
```

Comment:
Is there a special reason why it should be a slice of pointers? I would like to use only a simple list of strings.